ui/widgets/nuxeo-directory-suggestion.js

```diff
@@ -104,6 +105,11 @@ import { escapeHTML } from './nuxeo-selectivity.js';
          */
         dbl10n: { type: Boolean, value: false },
 
+        /**
+         * Checking this option means that the labels are localized with translations from JSON
+         * properties.
+        localize: { type: Boolean, value: true },
```

Comment:
This property is commented and that's why this code is working. We need to discuss the introduction of this API change first.

Reply:
I have fixed the unclosed comment